src/shared-components/PeopleTabUserEvents/PeopleTabUserEvents.tsx

```
@@ -0,0 +1,96 @@
+
+      {/* Event info */}
+      <div className={styles.peopleTabUserEventInfo}>
+        {eventName && <h4 className={styles.eventName}>{eventName}</h4>}
+        {eventDescription && (
+          <div>
+            <p className={styles.eventDescription}>{eventDescription}</p>
+
+            {actionName && (
+              <div className={styles.peopleTabUserEventAction}>
+                <button className={styles.peopleTabUserEventActionButton}>
+                  {actionIcon && (
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)